predelete/predelete.go

```diff
@@ -100,24 +101,31 @@ func PreDeleteEnterprisePortalSecret(clientset *kubernetes.Clientset) error {
 	return nil
 }
 
+// PreDeleteBootstrappingJobs deletes all jobs within the release namespace, that has specific label.
 func PreDeleteBootstrappingJobs(clientset *kubernetes.Clientset) error {
-	jobs, err := clientset.BatchV1().Jobs(data.AppConfig.TykPodNamespace).
+	jobs, err := clientset.
+		BatchV1().
+		Jobs(data.AppConfig.TykPodNamespace).
 		List(context.TODO(), metav1.ListOptions{})
```

Comment:
Can't we apply the label selector here as well?

Reply:
We have some kind of label check here: https://github.com/TykTechnologies/tyk-k8s-bootstrap/pull/16/files/8e3a749077a230f36fca7279b904dda02a9b663c#diff-f34eaecf68275b6e6734f692b11e1ba5115ab439cdd4c06fdb66a3c4e30f9913R116

In the first phase, while listing jobs, we do not know which jobs failed (`pre-install`, `post-install`?). That's why we first list all jobs within the release namespace. Then, if these jobs have a label key with `tyk.tyk.io/k8s-bootstrap`, we delete it

Reply:
I updated according to your comment. Can you please check again?